SIL.Scripture.Tests/VerseRefTests.cs

```diff
@@ -2067,6 +2067,24 @@ public void UnBridge()
 			Assert.AreEqual(new VerseRef("EXO 6:4"), new VerseRef("EXO 6:4-10").UnBridge());
 			Assert.AreEqual(new VerseRef("EXO 6:150monkeys"), new VerseRef("EXO 6:150monkeys").UnBridge());
 		}
+
+		/// <summary>
+		/// Tests the Verse property's set method with numerals from various Unicode-supported scripts
+		/// </summary>
+		[TestCase("५", 5, TestName = "Devanagari numeral")]
+		[TestCase("૧૬", 16, TestName = "Gujarati numeral")]
+		[TestCase("5",5, TestName = "Latin numeral")]
+		[TestCase("᠔", 4, TestName = "Mongolian numeral")]
+		[TestCase("A", -1, TestName = "Latin non-numeral")]
+		[TestCase("ะ", -1, TestName = "Thai non-numeral")]
+		public void SetVerse_InterpretUnicodeNumerals(string verseStr, int parsedVerse)
+		{
+			VerseRef vref = new VerseRef("EXO 6:1");
+
+			vref.Verse = verseStr;
+			Assert.AreEqual(parsedVerse, vref.VerseNum);
+		}
```

Comment:
:+1: 

Ahh, sorry, didn't notice that we test with different values.

This is fine as it is, but if you want you could use the ExpectedResult parameter:

```
[TestCase("ะ", ExpectedResult = -1, TestName = "Thai non-numeral")]
public int SetVerse_InterpretUnicodeNumerals(string verseStr)
{
	VerseRef vref = new VerseRef("EXO 6:1");

	vref.Verse = verseStr;
	return vref.VerseNum;
}
```

Reply:
I do like that slightly better. Thanks.